internal/user/user_manager.go

```diff
@@ -97,10 +99,30 @@ func GetUserProfile(ctx context.Context, r repo.Repository, username string) (*P
 	return p, nil
 }
 
+// all-lowercase subset of RFC 1123 without domain name (no dots allowed).
+var validHostnameRegexp = regexp.MustCompile(`^[a-z0-9]([a-z0-9\-]*[a-z0-9])*$`)
+
+// valid user is a superset of valid hostname (it allows _ and .)
+var validUserRegexp = regexp.MustCompile(`^[a-z0-9]([a-z0-9\-_.]*[a-z0-9])*$`)
```

Comment:
@pavannd1 PTAL

Reply:
Need to verify if this would break some existing assumptions. I'll take a look and update tomorrow 👍🏼

Reply:
FWIW we actually don't need to be as strict as the RFCs, basically just ensure no troublesome characters sneak in and all names are lowercase, so `[a-z0-9\-_.]+` might be sufficient for both.

Reply:
Okay, it would be good if we allow `_` and `.`. In the current model, we use dots in the generated hostname. I was looking to start using these commands to manage users and allowing dots will make the transition smoother.